packages/js/src/utils/callback-uri.ts

```diff
@@ -1,7 +1,7 @@
 import { QueryKey } from '../consts';
 import { LogtoError } from './errors';
 
-const parseUriParameters = (uri: string) => {
+export const parseUriParameters = (uri: string) => {
   const [, queryString = ''] = uri.split('?');
 
   return new URLSearchParams(queryString);
```

Comment:
per [MDN](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams), this class is web-worker only. let's add an issue and double check it later

Reply:
I'm afraid not, just checked Node.js documentation, and it's been there since v7.5
https://nodejs.org/api/url.html#class-urlsearchparams

Reply:
https://caniuse.com/?search=URLSearchParams

Reply:
Conclusion: It's safe to use URLSearchParams.
It is not Web Workers only API.

See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1644809199979179) on Slack.